searchlib/src/vespa/searchlib/engine/propertiesmap.h

```diff
@@ -111,21 +110,22 @@ class PropertiesMap
      *
      * @return cache properties
      **/
-    const search::fef::Properties &cacheProperties() const {
+    const Props &cacheProperties() const {
         return lookup(MapNames::CACHES);
     }
+    Props &cacheProperties() {
```

Comment:
This should not be overloaded since it alters inspection behavior based on object constness.